assets/app/styles/_core.less

```diff
@@ -899,6 +899,9 @@ h1 small.meta, .build-config-summary .meta {
 
 .ace_editor {
   border: 1px solid #999999;
+  .modal & {
+    border: 0;
+  }
```

Comment:
I'd rather have a class that adds the border style when we need it instead of special case when we take it away. Right now we have two places we want the border, view/browse/build-config.html and views/edit/build-config.html.